src/shared/install.c

```diff
@@ -1708,6 +1708,73 @@ static int install_info_discover_and_check(
         return install_info_may_process(ret ? *ret : NULL, paths, changes, n_changes);
 }
 
+int unit_file_verify_alias(const UnitFileInstallInfo *i, const char *dst, char **ret_dst) {
+        _cleanup_free_ char *dst_updated = NULL;
+        int r;
```

Comment:
would be good to have a comment here explaining what this does, and in particular saying that *ret_dst is initialized to NULL on success sometimes and when. That's not obvious...